src/balanceplatform/model_invalid_field.go

```diff
@@ -19,23 +19,23 @@ var _ common.MappedNullable = &InvalidField{}
 
 // InvalidField struct for InvalidField
 type InvalidField struct {
-	// Description of the validation error.
-	Message string `json:"message"`
```

Comment:
This change is breaking: the order of parameters is flipped, can we keep the order?

Reply:
Reverted